js/main.js

```diff
@@ -357,16 +358,24 @@ function handleDragEnd( event ) {
 		
 	//fade the element back in since it was faded it out and we need it again.
 	var draggingElement = $$('div.dragging')[0];
-	draggingElement.removeClass('dragging');
-	draggingElement.fade('in');
+
+	//Check to make sure the element being dragged wasn't disposed of. Like when added to the
+	//gearbag.
+	if( typeof draggingElement !== "undefined" ) {	
```

Comment:
You can do this comparison by value instead of type, which is a bit more concise:
`draggingElement !== undefined`